client/blocks/login/login-form.jsx

```diff
@@ -206,19 +194,21 @@ export class LoginForm extends Component {
 								<p>
 									{ this.props.translate(
 										'We found a WordPress.com account with the email address "%(email)s". ' +
-											'Log in to this account to connect it to your %(service)s profile.',
+										'Log in to this account to connect it to your %(service)s profile, ' +
+										'or choose a different %(service)s profile.',
 										{
 											args: {
 												email: this.props.socialAccountLinkEmail,
-												service: this.props.socialAccountLinkService,
+												service: capitalize( this.props.socialAccountLinkService ),
 											},
 										}
 									) }
 								</p>
 							</div>
 						) }
+
 						<label htmlFor="usernameOrEmail" className="login__form-userdata-username">
-							{ this.props.translate( 'Username or Email Address' ) }
+							{ this.props.translate( 'Email Address or Username' ) }
```

Comment:
:disappointed: 69 existing translations will be lost with this change.
If the string must change, the following option has already been translated **23** times:
`translate( 'Username or email address' )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/login-flow)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).